src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/Cohort/UpdatingDraftApprenticeship/WhenDraftApprenticeshipIsUpdated.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Linq;
+using AutoFixture;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Domain.Entities;
+using SFA.DAS.CommitmentsV2.Domain.Extensions;
+using SFA.DAS.CommitmentsV2.Models;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.UnitOfWork;
+
+namespace SFA.DAS.CommitmentsV2.UnitTests.Models.Cohort.UpdatingDraftApprenticeship
+{
+    public class WhenDraftApprenticeshipIsUpdated
```

Comment:
It looks like we have 3 fixtures with unit tests for updating a draft apprenticeship:

* https://github.com/SkillsFundingAgency/das-commitments/blob/CV-376-Approval-Reset-Defect/src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Domain/DraftApprenticeship/WhenDraftApprenticeshipIsUpdated.cs (acts on `DraftApprenticeship`)
* https://github.com/SkillsFundingAgency/das-commitments/blob/CV-376-Approval-Reset-Defect/src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/Cohort/WhenUpdatingDraftApprenticeship.cs (acts on `Cohort`)
* https://github.com/SkillsFundingAgency/das-commitments/blob/CV-376-Approval-Reset-Defect/src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/Cohort/UpdatingDraftApprenticeship/WhenDraftApprenticeshipIsUpdated.cs (acts on `Cohort`)

Is it worth reorganising these?